src/components/shared/Tooltip.js

```diff
@@ -82,25 +82,43 @@ export default class Tooltip extends React.PureComponent<Props, State> {
     const { children, mouseX, mouseY } = this.props;
     const { interiorElement } = this.state;
 
-    const offsetX = interiorElement
-      ? Math.max(0, mouseX + interiorElement.offsetWidth - window.innerWidth)
-      : 0;
+    // By default, position the tooltip below and at the right of the mouse cursor.
+    let top = mouseY + MOUSE_OFFSET;
+    let left = mouseX + MOUSE_OFFSET;
```

Comment:
I took the opportunity to calculate `top` and `left` directly intead of calculating the offset and applying it afterwards: I find it easier to reason with.